docs/tutorials/aws.md

```diff
@@ -125,6 +127,7 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRoleBinding
 metadata:
   name: external-dns-viewer
+  namespace: default
```

Comment:
The same comment for the `ClusterRole` applies to the `ClusterRoleBinding`, it is not namespaced.